server/midddleware/middleware.go

```diff
@@ -53,14 +56,19 @@ func Get(config *config.Config, tenantMgr *metadata.TenantManager, txMgr *transa
 	//
 	// Note: we don't add validate here and rather call it in server code because the validator interceptor returns gRPC
 	// error which is not convertible to the internal rest error code.
+	taggedLogger := log.Logger.With().
+		Str("env", tigrisconfig.GetEnvironment()).
+		Str("service", util.Service).
```

Comment:
Lovely idea! Let's do it!

Reply:
I might do it with another PR though as we will refactor this with @pboros further. This PR's main goal is to fix the traces and profiling info's incorrect env flags which it appears [to have done](https://us3.datadoghq.com/profiling/search?query=service%3Atigris-server%20env%3Adev&agg_m=%40prof_go_cpu_cores_by_top_methods.value&agg_q=%40prof_go_cpu_cores_by_top_methods.attributes.method&agg_t=avg&event=AQAAAYHzaz9Ia53hjwAAAABBWUh6YTBDV0FBQzRDd0xXRDlpV1lRQUE&my_code=disabled&profileId=AYHza0CWAAC4CwLWD9iWYQAA&viz=flame_graph&start=1657645063428&end=1657648663428&paused=false).

Adding the namespace would most likely require adding a custom middleware. I'll do that with a follow up PR so that we don't have to keep maintaining the other structural changes to Config et al!